package excepciones;

public class Main
{
	public static void main(String[] args)
	{
		int[] numeros = new int[10];

		String[] cadenas = new String[10];

		try
		{
			System.out.println("Hola");

			System.out.println(cadenas[0].length());

			numeros[100000] = 4;

			System.out.println("Adi√≥s");
		}
		catch(NullPointerException e)
		{

			System.out.println("Error.");
		}
	}
}
